nion/swift/Inspector.py

```diff
@@ -1865,7 +1889,6 @@ def change_display_type(item: typing.Tuple[str, typing.Optional[str]]) -> None:
     display_type_row.add_stretch()
     return display_type_row, listener
 
-
```

Comment:
There should be two blank lines before top-level functions and classes.

Reply:
This is done now - I also rebased to avoid having "Whitespace changes" commit messages